CommonTools/RecoAlgos/test/FKDTree_t.cpp

```diff
@@ -46,9 +46,11 @@ void TestFKDTree::test2D() {
     for (unsigned int i = 0; i < numberOfPointsOutsideTheBox; ++i) {
       float x = static_cast<float>(rand()) / (static_cast<float>(RAND_MAX));
       float y;
```

Comment:
It is probably irrelevant in the context of this unit test, but the `y` remains in fact undefined if `x` is outside `minX` and `maxX`...